build/source/engine/systemSolv.f90

```diff
@@ -2882,13 +2468,10 @@ f,                       & ! intent(out): new function value (mixed units)
   character(LEN=256)            :: cmessage                 ! error message of downwind routine
   ! variables for the line search
   REAL(DP), PARAMETER :: ALF=1.0e-4_dp,TOLX=epsilon(x),xTolInc=1.0e-4_dp
-  INTEGER(I4B) :: ndum,iterLS,iMax(1)
+  INTEGER(I4B) :: ndum,iterLS
```

Comment:
This code segment is a revision of the numerical recipes code (which did not follow summa conventions). I have deleted this routine (coming in a future PR) so will keep this soon-to-be-deleted code unchanged.